stages/org.osbuild.rhsm.facts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+"""
+This stage allows storing artifact-properties of the Red Hat Subscription
+Manager (rhsm-facts) in the built image.
+"""
+
+import json
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = r"""
+"properties": {
+  "facts": {
+    "type": "object",
+    "description": "Map of Red Hat Subscription Manager facts.",
+    "additionalProperties": true
+  }
+}
+"""
+
+
+def main(tree, options):
+    path = os.path.join(tree, "usr/share/osbuild/self")
```

Comment:
The commit message does not mention `self`, is this a left-over? FWIW, I would vote against `self` just because this doesn't *necessarily* describe just the image, it is just RHSM facts, whatever they might be.

Reply:
What would you use instead? "self" was discussed beforehand in this PR.

I think it kinda depends on what these facts define. "self" was picked as a subdir in "osbuild" to distinguish data describing the host image from data installed by rpm as part of osbuild to build new images on this host.

This is not so much about what the data describes, but rather where the data comes from. In this case, it states the data was part of the host-image, rather than part of an installed osbuild-rpm.

Reply:
Ah, that makes sense. I misunderstood and missed the discussion.